crates/core/app/src/app/mod.rs

```diff
@@ -51,6 +51,9 @@ pub mod state_key;
 /// The inter-block state being written to by the application.
 type InterBlockState = Arc<StateDelta<Snapshot>>;
 
+/// The maximum size of a CometBFT block payload.
+const MAX_BLOCK_PAYLOAD_SIZE_BYTES: u64 = (1 << 20) - 1;
```

Comment:
What happens if a chain votes to raise block `max_bytes` to a value larger than 1MiB? This hardcoded value just doesn't allow larger blocks, right?

Reply:
That's right, we don't have a flow to change consensus parameters anyway